examples/v0/check.py

```diff
@@ -0,0 +1,28 @@
+from authzed.api.v0 import CheckRequest, Client, ObjectAndRelation, User
+from grpcutil import bearer_token_credentials
+
+
+emilia = User("blog/user", "emilia")
```

Comment:
Have the prefix come from a variable and then add a comment indicating it needs to be replaced

Reply:
I think that just adds more noise to the example. The intent behind the examples is to let people see the API and not show them the most succinct or elegant usage of it.

Reply:
As discussed, okay for now